api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -543,6 +543,14 @@ message HttpConnectionManager {
   // emitted by the connection manager.
   repeated config.accesslog.v3.AccessLog access_log = 13;
 
+  // The interval to flush the above access logs. By default, the HCM will flush exactly one access log
+  // when the HTTP request is complete. If this field is set, the HCM will flush access logs periodically
+  // at the specified interval. This is especially useful in the case of long-lived requests, such as
+  // CONNECT and Websockets.
+  // The interval must be at least 1 second.
+  google.protobuf.Duration access_log_flush_interval = 53
```

Comment:
I think it would be useful to log on stream open/close as well.

Reply:
stream close is already logged via the normal process, right?

Reply:
Yeah, but it's not clear from the description. Stream open is certainly useful for long running operations.

Reply:
This was supported in the TCP proxy. It would be better to keep both fields have similar behavior.
https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto

Reply:
Added comment clarifying the stream close, as well as adjusted the min duration to 1ms to align w/ TCP Proxy.